### ISSUE TITLE
[feature request] color theme button

### ISSUE THREAD

**Issue by sudo-Tiz** [penguin86/nextcloud-maps-client#28]
Could it be possible to have a button to select dark/light theme so we can enable android dark theme globally but still have a light themed map ?

**Comment by StrikerTwo** [comment]
Yes please. The purple map just looks ugly, and my tablet on light mode gives me eye cancer.
